### ISSUE TITLE
Update example manifests to work with new "phased"! execution flow

### ISSUE THREAD

**Issue by zanete** [Green-Software-Foundation/if#812]

**Comment by zanete** [comment]
@jmcook1186 please add an example of the manifest changes

**Comment by zanete** [comment]
@jmcook1186 @narekhovhannisyan can you confirm please that this is something that would be useful to do now/soon and that @MariamKhalatova is in a good position to do this? 🙏

**Comment by jmcook1186** [comment]
yes this will need to be applied across our entire set of manifests (all subdirectories inside `if/manifests`)

**Comment by zanete** [comment]
a few manifests have not updated, @manushak will help identify which ones. @MariamKhalatova to sort out once clarified 🙏

**Comment by manushak** [comment]
these manifests need to be updated
- manifests/examples/pipelines/outputs-if-diff/pipeline-with-aggregate.yaml
- manifests/examples/pipelines/outputs-if-diff/pipeline-with-mocks.yaml
